src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
+  const longtermDiscount = 50;
+  const shorttermDiscount = 20;
+
+  if (days >= 7) {
+    result = result - longtermDiscount;
```

Comment:
just use,

`return totalPrice - longTermDiscount`

don't need to reassign value to variable